ui/main.qml

```diff
@@ -187,6 +187,11 @@ StatusWindow {
                 Style.changeTheme(localAppSettings.theme, systemPalette.isCurrentSystemThemeDark())
                 Style.changeFontSize(localAccountSensitiveSettings.fontSize)
                 Theme.updateFontSize(localAccountSensitiveSettings.fontSize)
+            } else if(state === Constansts.appState.appEncryptionProcess) {
+                loader.sourceComponent = undefined
+                appLoadingAnimation.active = true
+                appLoadingAnimation.item.splashScreenText = qsTr("Loading Status... Please DO NOT CLOSE the app.")
```

Comment:
I'd be clearer with what is happening. Something like:
"Database re-encryption in progress. Please do NOT close the app."
With a sub-title:
"This may take up to 30 minutes.Sorry for the inconvenience. This process is a one time thing and is necessary for the proper functioning of the application"

Reply:
Yeah, some "better" wording is needed here

Reply:
<img width="1201" alt="Снимок экрана 2023-03-29 в 17 43 47" src="https://user-images.githubusercontent.com/82511785/228576085-a4b1599a-7d1a-40c2-a4aa-eface7619b0a.png">

is it okay?

Reply:
Did you forget to push this update @borismelnik ?

Reply:
im not forget - I was waiting for approve :) one sec